### DISCUSSION TITLE
Working with Radiomaster TX12

### DISCUSSION THREAD

**Discussion by alex2root** [RomanLut/hx_espnow_rc#25] [General]
Good afternoon.

I have a Radiomaster TX12 with EdgeTX firmware.

I only need the S-Bus channel, without telemetry.

Can I use your project?

**Comment by RomanLut** [comment]
Yes you can, you have to build transmitter module similar to:https://github.com/RomanLut/hx_espnow_rc/blob/main/doc/tx_external_module.md but for JR module bay:
![image](https://github.com/user-attachments/assets/35b05226-734b-469c-97e1-c23995f54843)
and you may skip bluetooth module installation if you do not need telemetry.
